examples/terraform/etcd3-install/README.md

```diff
@@ -35,6 +35,16 @@ matchbox_rpc_endpoint = "matchbox.example.com:8081"
 ssh_authorized_key = "ADD ME"
 ```
 
+##### OEM Customisation
```

Comment:
Maybe we carve out a place to have docs for individual profiles. As more example clusters are added, we probably don't want to duplicate. Though I appreciate that you've written docs.